src/app/app-routing.module.ts

```diff
@@ -72,11 +70,11 @@ const routes: Routes = [
         path: 'archive',
         component: ArchiveComponent
       },
-      {
-        path: 'my-profile',
-        component: MyProfileComponent,
-        canActivate: [AuthGuard]
-      },
+      // {
+      //   path: 'my-profile',
```

Comment:
You removed the route 'my-profile' but you still navigate to it in various different places - is that intended?

Reply:
My bad - I will fix that.